pkg/utils/utils.go

```diff
@@ -35,7 +43,12 @@ func AskForData(config map[string]*project.ServiceConfig) {
 			msg := fmt.Sprintf("[%s] No ports defined to send traffic to, no provider service will be created. Do you want to create a service? y/[n]: ", svc)
 			if resp := getInput(msg); resp == "y" {
 				msg = fmt.Sprintf("[%s] Enter ports: ", svc)
```

Comment:
How about something like this:
Either specify both ports (HOST:CONTAINER), or just the container port (a random host port will be chosen). (stolen from docker-compose format  [reference](https://docs.docker.com/compose/compose-file/#/ports)) You can specify specify multiple ports separated by space.

Or we can just put example here.
Example: 8080:80 8443:443